src/ripple/app/misc/impl/ValidatorSite.cpp

```diff
@@ -173,7 +173,7 @@ void
 ValidatorSite::makeRequest (
     std::shared_ptr<Site::Resource> resource,
     std::size_t siteIdx,
-    std::lock_guard<std::mutex>& lock)
+    std::unique_lock<std::mutex> const& /*lock*/)
```

Comment:
I'd remove the commented out name. C++ supports nameless arguments for a reason, and the intent behind a `std::unique_lock` passed by `const&` is pretty clear. The name doesn't really add anything.